examples/compute/Holmakefile

```diff
@@ -0,0 +1,11 @@
+CLINE_OPTIONS = -r
+
+INCLUDES = $(HOLBADIR)/examples/compute/src/theory \
+					 $(HOLBADIR)/examples/compute/test \
+					 $(HOLBADIR)/examples/compute/examples \
```

Comment:
Could we make running the tests a task in this `Holmakefile`? See https://github.com/kth-step/HolBA/blob/master/examples/riscv/Holmakefile

Reply:
Sorry I had only made it available inside the `examples/compute/examples/`. https://github.com/kth-step/HolBA/pull/187/commits/83c77721111cca5ef28f79a2bc34bddfd340f6b5 should fix this

Reply:
Hmmm I’m sorry I misunderstood (again…). You meant the tests should be run from the `Holmakefile`, not the examples (Could be both obviously). 
I’ll check what I can do regarding tests

Reply:
@jules-timmerman building the `examples/*` stuff is already done via the `-r` CLINE_OPTION. It's fine that the examples are in their own `*Script.sml` files and get built like HOL4 theories.

However, I argue that small tests should not be in `*Script.sml` files, but in regular files called something like `test-stuff.sml`, `test-more-stuff.sml`, that live in a relevant directory, which could here be `$(HOLBADIR)/examples/compute`. This ensures they [the tests] are not persisted as theories and can be run from scratch again and again via `Holmake test`.

Reply:
Recall that as long as you open HOL4 core modules:
```sml
open HolKernel boolLib Parse bossLib;
```
you can define theorems and the like using the `prove` function` or `Theorem my_local_thm[local] ...`, without being inside a theory.

If you need some `Definition` to define the local theorem, I argue this definition should live in some non-test HOL4 theory and `open`ed in the `test-*.sml` file.

Does that make it clearer?

Reply:
Okay I understand now thank you very much for your explanation. I’ll try changing that. Sorry for the ugly git history

Reply:
Please feel free to make as many commits as you want, I think we will squash PR to make it a single commit in the HolBA history. The private development repo can have the development history if we need it.

Reply:
Okay I moved the tests in a single file in https://github.com/kth-step/HolBA/pull/187/commits/5c858d2c1019548f6c74fd74ff33ccd9b82fca6e. I think this should be what you had in mind. I’ll add some more unit tests but I don’t think I’ll have a lot so that’s why I only made a single file

Reply:
Looks good to me, thanks!